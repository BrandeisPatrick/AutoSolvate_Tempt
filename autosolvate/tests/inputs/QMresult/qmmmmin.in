gpr QMMM Min
 &cntrl
  imin   = 1,
  maxcyc = 250,
  irest  = 0, ! 0- new simulation 1- restart
  ntx    = 1, ! 1-read in coordinates, but not velocity, 5-both
  cut    = 8.0,
  ig     = -1, !random seed
  ntb    = 1,
  ntpr   = 1, !print detials to log every step
  ntwx   = 1, !write coordinates to mdcrd every step
  ntwr   = 1, !write restart file every step
  ifqnt  = 1,!turn on QM/MM
/
 &qmmm
  qmmask    = ':1',
  qmcut     = 8,
  qmcharge  = 0,
  qmshake   = 0, !Shake QM H atoms if shake is turned on (NTC>1) (default)
  qm_theory = 'EXTERN',
  qm_ewald  = 0,
  qmgb      = 0,
  verbosity = 2,
  writepdb  = 1,
 /
 &tc
  executable = 'terachem',
  use_template = 1,
 /
