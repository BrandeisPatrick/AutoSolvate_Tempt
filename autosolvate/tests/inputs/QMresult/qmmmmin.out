
          -------------------------------------------------------
          Amber 22 SANDER                              2022
          -------------------------------------------------------

| Run on 03/19/2024 at 19:26:07

|   Executable path: sander
| Working directory: /home/sunnyxun/projects/new_project_AutoMCPB/RedoxPotential/Fe/plus2/FFgen
|          Hostname: node7

  [-O]verwriting output

File Assignments:
|  MDIN: qmmmmin.in                                                            
| MDOUT: qmmmmin.out                                                           
|INPCRD: mm.ncrst                                                              
|  PARM: Fe_plus2_solvated.prmtop                                              
|RESTRT: qmmm.ncrst                                                            
|  REFC: refc                                                                  
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: Fe_plus2_solvated-qmmmmin.netcdf                                      
|MDINFO: qmmmmin.info                                                          
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

gpr QMMM Min                                                                   
 &cntrl                                                                        
  imin   = 1,                                                                  
  maxcyc = 250,                                                                
  irest  = 0, ! 0- new simulation 1- restart                                   
  ntx    = 1, ! 1-read in coordinates, but not velocity, 5-both                
  cut    = 8.0,                                                                
  ig     = -1, !random seed                                                    
  ntb    = 1,                                                                  
  ntpr   = 1, !print detials to log every step                                 
  ntwx   = 1, !write coordinates to mdcrd every step                           
  ntwr   = 1, !write restart file every step                                   
  ifqnt  = 1,!turn on QM/MM                                                    
/                                                                              
 &qmmm                                                                         
  qmmask    = ':1',                                                            
  qmcut     = 8,                                                               
  qmcharge  = 0,                                                               
  qmshake   = 0, !Shake QM H atoms if shake is turned on (NTC>1) (default)     
  qm_theory = 'EXTERN',                                                        
  qm_ewald  = 0,                                                               
  qmgb      = 0,                                                               
  verbosity = 2,                                                               
  writepdb  = 1,                                                               
 /                                                                             
 &tc                                                                           
  executable = 'terachem',                                                     
  use_template = 1,                                                            
 /                                                                             

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
 getting box info from netcdf restart file
| NetCDF restart box info found
|Largest sphere to fit in unit cell has radius =    27.781
| New format PARM file being parsed.
| Version =    1.000 Date = 03/19/24 Time = 15:21:12
 NATOM  =   10953 NTYPES =       8 NBONH =    5469 MBONA  =    3675
 NTHETH =   10938 MTHETA =    1890 NPHIH =    5547 MPHIA  =     144
 NHPARM =       0 NPARM  =       0 NNB   =   29423 NRES   =    1821
 NBONA  =    3675 NTHETA =    1890 NPHIA =     144 NUMBND =      26
 NUMANG =      66 NPTRA  =       6 NATYP =      16 NPHB   =       0
 IFBOX  =       1 NMXRS  =      20 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0


|     Memory Use     Allocated
|     Real             1632430
|     Hollerith          34682
|     Integer           629189
|     Max Pairs        3651000
|     nblistReal        131436
|     nblist Int        695998
|       Total            33353 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
LOADING THE QUANTUM ATOMS AS GROUPS
     Mask :1; matches     1 atoms

     BOX TYPE: RECTILINEAR
Note: ig = -1. Setting random seed to   854509 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       2, ntpr    =       1, ntrx    =       1, ntwr    =       1
     iwrap   =       0, ntwx    =       1, ntwv    =       0, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       1, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =   8.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Energy minimization:
     maxcyc  =     250, ncyc    =      10, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       1, eedmeth =       1, netfrc  =       0
     Box X =   55.561   Box Y =   55.561   Box Z =   55.561
     Alpha =   90.000   Beta  =   90.000   Gamma =   90.000
     NFFT1 =   60       NFFT2 =   60       NFFT3 =   60
     Cutoff=    8.000   Tol   =0.100E-04
     Ewald Coefficient =  0.34864
     Interpolation order =    4

QMMM options:
             ifqnt = True       nquant =        1
              qmgb =        0  qmcharge =        0   adjust_q =        2
              spin =        1     qmcut =   8.0000    qmshake =        0
          qmmm_int =        1
     lnk_atomic_no =        1   lnk_dis =   1.0900 lnk_method =        1
          qm_theory =     EXTERN verbosity =        2
            qmqmdx = Analytical
          qm_ewald =        0 qm_pme = False 
|  INFO: Old style inpcrd file read


--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =   620.000 ps

 Number of triangulated 3-point waters found:        0

     Sum of charges from parm topology file =   0.01815169
     Assuming uniform neutralizing plasma
QMMM: ADJUSTING CHARGES
QMMM: ----------------------------------------------------------------------
QMMM: adjust_q = 2
QMMM: Uniformly adjusting the charge of MM atoms to conserve total charge.
QMMM:                             qm_charge =    0
QMMM: QM atom RESP charge sum (inc MM link) =    0.193
QMMM: Adjusting each MM atom resp charge by =    0.000
QMMM:          Sum of MM + QM region is now =    0.018
QMMM: ----------------------------------------------------------------------
 ---------------------------------------------------
 APPROXIMATING switch and d/dx switch using CUBIC SPLINE INTERPOLATION
 using   5000.0 points per unit in tabled values
 TESTING RELATIVE ERROR over r ranging from 0.0 to cutoff
| CHECK switch(x): max rel err =   0.2738E-14   at   2.422500
| CHECK d/dx switch(x): max rel err =   0.8332E-11   at   2.782960
 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =    1435247
| TOTAL SIZE OF NONBOND LIST =    1435247
 QMMM: No. QMMM Pairs per QM atom:          150

QMMM: Link Atom Information
QMMM: ------------------------------------------------------------------------
QMMM:  nlink =     6                   Link Coords              Resp Charges
QMMM:    MM(typ)  -  QM(typ)      X         Y         Z         MM        QM
QMMM:     2 Y7        1 M1       0.886     0.351     0.530     0.024    -0.007
QMMM:    13 Y8        1 M1       0.082    -0.772     0.765     0.035    -0.007
QMMM:    22 Y9        1 M1       0.549    -0.667    -0.664     0.032    -0.007
QMMM:    33 Z1        1 M1      -0.841    -0.423    -0.549     0.030    -0.007
QMMM:    42 Z2        1 M1      -0.012     0.826    -0.711     0.039    -0.007
QMMM:    53 Z3        1 M1      -0.623     0.705     0.550     0.040    -0.007
QMMM: ------------------------------------------------------------------------

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      Fe       0.0000   -0.0000    0.0000
  QMMM:     2              *H        0.8857    0.3511    0.5295
  QMMM:     3              *H        0.0819   -0.7718    0.7653
  QMMM:     4              *H        0.5491   -0.6672   -0.6643
  QMMM:     5              *H       -0.8411   -0.4232   -0.5491
  QMMM:     6              *H       -0.0123    0.8262   -0.7109
  QMMM:     7              *H       -0.6232    0.7052    0.5499

--------------------------------------------------------------------------------
  3.1 QM CALCULATION INFO
--------------------------------------------------------------------------------

| Please also cite the following work for the use of the QM/MM interface:

| A. W. G"otz, M. A. Clark, R. C. Walker
| "An extensible interface for ab initio QM/MM molecular dynamics simulations
|  with AMBER"
| J. Comput. Chem. 35 (2014) 95-108.
| DOI: 10.1002/jcc.23444

| C. M. Isborn , A. W. G"otz , M. A. Clark , R. C. Walker, T. J. Martinez
| "Electronic Absorption Spectra from MM and ab initio QM/MM Molecular Dynamics:
|  Environmental Effects on the Absorption Spectrum of Photoactive Yellow Protein"
| J. Chem. Theory Comput. 8 (2012) 5092-5106
| DOI: 10.1021/ct3006826

 Constants for unit conversion taken from
 Mohr, Taylor, Newell, Rev. Mod. Phys. 80 (2008) 633-730
 and using the thermochemical calorie (1 cal = 4.184 J):

 A_TO_BOHRS  =  1.889726132873E+00
 AU_TO_KCAL  =  6.2750946943E+02
 AU_TO_DEBYE =  2.54174623E+00

   >>> Running QM calculation with TeraChem <<<

 SANDER BOMB in subroutine check_installation (qm2_extern_util_module)
 Executable "terachem" not found
 Please check that "terachem" is in your path
